src/common/comp_tests/check_method_config.py

```diff
@@ -9,13 +9,21 @@
 
 ## VIASH END
 
+NAME_MAXLEN = 50
+
+SUMMARY_MAXLEN = 400
+
+DESCRIPTION_MAXLEN = 1000
+
+
 print("Load config data", flush=True)
 with open(meta["config"], "r") as file:
                 config = yaml.safe_load(file)
 
 
 print("Check general fields", flush=True)
 assert "name" in config["functionality"] is not None, "Name not a field or is empty"
```

Comment:
I think this assertion is unneeded, since Viash will already complain if the .functionality field is not defined.

```suggestion
```